docs/resiliency/index.asciidoc

```diff
@@ -56,6 +56,14 @@ If you encounter an issue, https://github.com/elastic/elasticsearch/issues[pleas
 We are committed to tracking down and fixing all the issues that are posted.
 
 [float]
+==== Jepsen Tests
+
+The Jepsen platform is specifically designed to test distributed systems. It is not a single test and is regularly adapted
+to create new scenarios. We have taken over the scenarios tested so far and added them to our testing infrastructure. Of course
+as the system evolves, new scenarios can come up that are not yet covered. We are committed to investigate them and report the issues
```

Comment:
"We are committed to investigating all new scenarios and will report issues that we find on this page and in our GitHub repository."